ddtrace/span.py

```diff
@@ -169,6 +169,17 @@ def start(self, value):
         # type: (Union[int, float]) -> None
         self.start_ns = int(value * 1e9)
 
+    @property
+    def resource(self):
+        return self._resource[0]
+
+    @resource.setter
+    def resource(self, value):
+        self._resource[0] = value
+
+    def resource_container(self):
```

Comment:
we won't want to make this public, so internal/profiling should just access as `span._resource`

otherwise we have to support this API for customers as well.

Reply:
aha, ok!